src/api/submissions/submissions.service.js

```diff
@@ -100,7 +101,7 @@ const getRegistrationMetaData = async (councilCode) => {
       "getRegistrationMetaData"
     );
     return {
-      "fsa-rn": fsa_rn ? fsa_rn["fsa-rn"] : undefined,
+      "fsa-rn": fsa_rn ? fsa_rn["fsa-rn"] : "tmp_" + uuid.v4(),
```

Comment:
Does this need to check fsa_rn["fsa-rn"] is not undefined?

Reply:
👍